digitalpalireader/content/js/web/io.js

```diff
@@ -0,0 +1,393 @@
+function setI() {
+	i = 1;
+}
+
+ioCheck = true;
+
+
+function readFile(aFileKey)
+{
+  console.log("History not yet implemented.");
```

Comment:
is this the only change in the file? it looks fine. If there are others let me know so i can check those as well.

also can you please do an Alt-Shift-F if you are using VScode - that will normalize all the tabs to spaces. so subsequent changes will not show that conversion.

Reply:
Yes it's the only change. Thanks for the hint with Alt-Shift-F.